frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/common.embedding.unit.spec.tsx

```diff
@@ -83,6 +83,45 @@ describe("[OSS] embedding settings", () => {
       });
     });
 
+    describe("interactive embedding", () => {
```

Comment:
The diff looks a bit confusing, but what I do here is to move `describe("embedding SDK", () => {` into its own block that uses `enable-embedding-sdk` instead of `enable-embedding`.

So the tests below are the same tests.